release-schema.json

```
@@ -84,6 +84,24 @@
             "null"
           ],
           "minLength": 1
+        },
+        "appliesTo": {
+          "title": "Applies to",
+          "description": "Whether the criterion applies to suppliers, subcontractors, or both.",
+          "type": [
+            "array",
+            "null"
+          ],
+          "items": {
+            "type": "string",
+            "enum": [
+              "supplier",
+              "subcontractor"
+            ]
+          },
+          "codelist": "appliesTo.csv",
+          "openCodelist": false,
+          "omitWhenMerged": true
```

I think we want uniqueItems. Not sure why omitWhenMerged is here.

```suggestion
          "uniqueItems": true
```